lib/room.rb

```diff
@@ -1,12 +1,25 @@
-class Rom
-
+class Room
+attr_reader :category, :painted
```

Comment:
since we have a method `is_painted?` that gives us the value of `@painted`, we don't need this attr_reader.